src/placeos/api_wrapper/users.cr

```diff
@@ -11,6 +11,78 @@ module PlaceOS
 
     getter base : String = "#{API_ROOT}/users"
 
+    # CRUD Actions
+    def search(
+      q : String? = nil,
+      limit : Int = 20,
+      offset : Int = 0
+    )
+      get base, params: from_args, as: Array(User)
+    end
+
+    def create(
+      authority_id : String,
+      name : String,
+      nickname : String?,
+      email : String?,
+      phone : String?,
+      country : String?,
+      image : String?,
+      ui_theme : String?,
+      metadata : String?,
+      login_name : String?,
+      staff_id : String?,
+      first_name : String?,
+      last_name : String?,
+      building : String?,
+      password_digest : String?,
+      email_digest : String?,
+      card_number : String?,
+      deleted : Bool?,
+      groups : Array(String)?,
+      access_token : String?,
+      refresh_token : String?,
+      expires_at : Int64?,
+      expires : Bool?,
+      password : String?,
+      sys_admin : Bool?,
+      support : Bool?
+    )
+      post base, body: from_args, as: User
+    end
+
+    def update(
+      id : String,
```

Comment:
```suggestion
```

you can't update the id of a model

in fact, a lot of these cannot be updated.

Reply:
the `id` is not used to update the model, it is used to `put #{base}/#{id}`

Reply:
lol my bad